### DISCUSSION TITLE
Where to download Plugins & Themes

### DISCUSSION THREAD

**Discussion by codeplay-code** [BoidCMS/BoidCMS#33] [Q&A] [answered]
Pluging e Themes download?

**Comment by sysadev** [comment]
Hello @marcocodeplay,

To download themes please visit here: https://boidcms.github.io/#/themes/
For plugins: https://boidcms.github.io/#/plugins/

Please know that some of the plugins are still not available for download. But I'll work on it soon.

**Reply by codeplay-code** [reply, marked as answer]
Ok ty Bro
